app/controllers/stories_controller.rb

```diff
@@ -400,6 +396,15 @@ def set_article_json_ld
     }
   end
 
+  def seo_optimized_images
+    # This array of images exists for SEO optimization purposes.
+    # For more info on this structure, please refer to this documentation:
+    # https://developers.google.com/search/docs/data-types/article
+    [ApplicationController.helpers.article_social_image_url(@article, width: 1080, height: 1080),
```

Comment:
Is it possible that we'll ever want to build this array for other articles, too? Right now, we're just always using `@article` here, but if we wanted to make this more reusable, we could allow someone to pass in an article, and then use that.

Not sure if we are going to want to build this kind of SEO-optimized array of images anytime soon, but if we do, this would definitely make that easier.

Something else to consider (not blocking): if we _do_ expect that we will want to build this array of variously-sized images in other places, maybe we don't want it to live in this controller, but rather, in a more reusable/shared location. Just a question in the back of my mind after looking at this code :)

Reply:
These are good questions that I don't have great answers to. 😅 I could potentially see us using this array for other articles in the future, but I don't know to what extent and I don't know if refactoring this code and pulling it out into another controller is out of scope for this currently.

Does anyone else have any other thoughts or opinions on this? 😅

Reply:
It's a good idea to extract for reusability but I think we can do it when we're applying SEO optimization else too.

Reply:
I just opened an issue (https://github.com/thepracticaldev/dev.to/issues/7495) and added it to my project board that addresses possible generalization of this image array in addition to refactoring the current JSON-LD code (probably during cool-down). 😊

Reply:
I agree with postponing extracting it until we have our next use case.

Reply:
This all sounds good to me; again, I definitely don't think any of this should block this PR!